portfolio/src/main/webapp/index.html

```diff
@@ -10,54 +10,104 @@
     <div class="all-content" id="content">
         <table id="allTable" class="table-allInfo">
             <tr>
-                <td id="name">David Vargas Puga</td>
-            </tr>
-            <tr>
-                <td id="aboutMe">About Me</td>
+                <td id="avi">
```

Comment:
As much as possible, try to avoid adding id or class attributes if they are not being used in either your JS or CSS code.

This makes it easier to track down what is modifying certain UI elements when debugging issues in the future.

I believe #avi is not currently used, so we can remove it from this HTML element. If you plan on using this id later on, feel free to add it back once you have JS/CSS code that uses it.